src/quanguru/classes/QGates.py

```diff
@@ -79,7 +80,7 @@ def _rotMat(self, collapseOps = None, decayRates = None, openSys=False): #pylint
                 flipOpN = operators.compositeOp(rotOp(sys[i+1].dimension, isDim=True),
                                                 sys[i+1]._dimsBefore, sys[i+1]._dimsAfter)
                 flipUn = evolution.Unitary(self.phase*self.angle*flipOpN) @ flipUn
-            self._paramBoundBase__matrix = evolution._prepostSO(flipUn) if (openSys or isinstance(collapseOps, list) or self._isOpen) else flipUn # pylint: disable=assigning-non-slot,line-too-long,protected-access
+            self._paramBoundBase__matrix = evolution._prepostSO(operatorA = flipUn, operatorB = hc(flipUn)) if (openSys or isinstance(collapseOps, list) or self._isOpen) else flipUn # pylint: disable=assigning-non-slot,line-too-long,protected-access
```

Comment:
```suggestion
            self._paramBoundBase__matrix = evolution._prepostSO(operatorA = flipUn, operatorB = linearAlgebra.hc(flipUn)) if (openSys or isinstance(collapseOps, list) or self._isOpen) else flipUn # pylint: disable=assigning-non-slot,line-too-long,protected-access
```

Reply:
nice catch.
For my open Heisenberg DQS simulations, I was only using `xGate` with $angle = \frac{\pi}{2}$, therefore `flipUn` was $i\sigma_{x}$, whose `hc` is $-i\sigma_{x}$. This means that the effect of this bug on my simulations is a global phase. But, since I apply the these `xGate` for even number of times, this global phase cancels out.
Do you agree? if so, I am curious how did you found out this mistake?

Reply:
I agree that the global phase cancels out.
I don't remember exactly, but I think I tested every step individually using Quantum Toolbox, and rotation operators were producing matrices with a minus sign as you described. 

Do you want me to write a unit test as well?